app/controllers/curation_concerns/curation_concerns_controller.rb

```diff
@@ -43,7 +54,15 @@ def presenter
     end
 
     def manifest_builder
-      ManifestBuilder.new(presenter, ssl: request.ssl?)
+      ManifestBuilder.new(presenter, ssl: request.ssl?, login_url: login_url, logout_url: logout_url)
```

Comment:
That sounds good -- it would detangle ManfiestBuilder and AuthManifestBuilder, and let us plug in more services in the controller if that makes sense later.